src/booleans.js

```diff
@@ -1,61 +1,127 @@
 function negate(a) {
-  // your code here
+  if(a === true){
+    return false;
+  } else if(a === false){
+    return true;
+  }
 };
 
 function both(a, b) {
-  // your code here
+ if ((a === true) && (b === true)){
+   return true;
+ } else {
+   return false;
+ }
 };
 
 function either(a, b) {
-  // your code here
+  if ((a === true) || (b === true)){
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function none(a, b) {
-  // your code here
+  if ((a !== true) && (b !== true)){
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function one(a, b) {
-  // your code here
+  if ((a === true) && (b === false)){
+    return true;
+  } else if ((a === false) && (b === true)) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function truthiness(a) {
-  // your code here
+  if (a){
+    return true;
+  } else{
+    return false;
+  }
 };
 
 function isEqual(a, b) {
-  // your code here
+  if (a === b){
+    return true;
+  } else {
+    return false;
+  };
 };
 
 function isGreaterThan(a, b) {
-  // your code here
+  if (a > b){
+    return true;
+  } else{
+    return false;
+  };
 };
 
 function isLessThanOrEqualTo(a, b) {
-  // your code here
+  if (a <= b){
+    return true;
+  } else{
+    return false;
+  };
 };
 
 function isOdd(a) {
-  // your code here
+  if ((a % 2) > 0){
+    return true;
+  } else{
+    return false;
+  };
 };
 
 function isEven(a) {
-  // your code here
+  if ((a % 2) === 0){
+    return true;
+  } else{
+    return false;
+  }
 };
 
 function isSquare(a) {
-  // your code here
+  squareRoot = Math.sqrt(a);
```

Comment:
If you are defining a variable, try to always use `const` or `let` as it is best practice.

Reply:
You could also simplify overall. Number.isInteger() returns a boolean already, so you don't need to compare it to true. You could write something like `return Number.isInteger(Math.sqrt(a));`